### ISSUE TITLE
Str() won't count leading zeroes when calculating default width for large values

### ISSUE THREAD

**Issue by vszakats** [harbour/core#98]
```Harbour
? RTSTR( 00000500000000000000 )    
? RTSTR( 0000500000000000000 )     
? RTSTR( 000500000000000000 )      
? RTSTR( 00500000000000000 )       
? RTSTR( 0500000000000000 )        

FUNCTION RTSTR( nValue )
   RETURN "|" + Str( Len( Str( nValue ) ), 3 ) + " " + Str( nValue ) + "|"
```

Clipper 5.2e/5.3b:
```
| 21       500000000000000|
| 20      500000000000000|
| 19     500000000000000|
| 18    500000000000000|
| 17   500000000000000|
```

Harbour:
```
| 16  500000000000000|
| 16  500000000000000|
| 16  500000000000000|
| 16  500000000000000|
| 16  500000000000000|
```

**Comment by druzus** [comment]
It's a known issue discussed many times few years ago.
Above is side effect of introducing 64bit integers in generated PCODE.
Integers in PCODE does not have length attribute.
I created macros in hbdefs.h which can force Cl*pper comptible behavior
by replacing 64bit integers in PCODE with double values like in Cl*pper
(see PCODE_LONG_LIM() macro) anyhow the bad side effect is lost
precision (IEEE758 double value has only 53bit presission so 10bits are
lost).
I also proposed as real solution adding new PCODE for 64bit integers
with SIZE attribute if it's real problem but AFAIR users and other developers
ignored it. So I only left the test in HBTEST just to show it's real incompatibility
though minor.
